packages/test-snaps/LICENSE

```diff
@@ -0,0 +1,15 @@
+ISC License
```

Comment:
Should this be ISC licensed?

Reply:
Probably? It's not really intended to be used as example.

Reply:
Test dapp is MIT licensed, perhaps we should follow that? 🤔